cli/Cargo.toml

```diff
@@ -29,7 +29,7 @@ maplit = "1.0.2"
 minisign-verify = "0.1.8"
 nom = "6.1.2"
 open = "2.0.1"
-phylum_types = { git = "https://github.com/phylum-dev/phylum-types", branch = "master" }
+phylum_types = { path = "/home/chris/programming/rust/phylum-types" }
```

Comment:
I've added the `UserInfo` to the `phylum-types` locally, since it seems like that's the correct location for HTTP request types. I'll send a PR for that if the approach taken in this PR seems reasonable.